cloverage/test/cloverage/test_report.clj

```diff
@@ -1,7 +1,30 @@
 (ns cloverage.test-report
   (:import java.io.File)
   (:use clojure.test
-        cloverage.report))
+        cloverage.report)
+  (:require [clojure.tools.reader :as r]
+            [clojure.java.io :as io]))
+
+(defn _parse-readable
+  "parse in all forms from reader"
+  [reader]
+  (with-open [pb-reader (java.io.PushbackReader. reader)]
+    (binding [*read-eval* false]
+      (let [exp-reader (repeatedly #(read pb-reader false nil))]
+        (doall (take-while identity exp-reader))))))
+
+(defn _get-resource-as-stream
+  "open resource from classpath as stream"
+  [resource]
+  (.getResourceAsStream (clojure.lang.RT/baseLoader) resource))
```

Comment:
Are the prefix underscores intended to mark this as private? If so, please do not use a prefix underscore and instead use the `^:private` metadata tag (or, in my opinion less preferably, `def-` / `defn-`).